pkg/proto/pbgo/core/api.pb.go

```diff
@@ -1,20 +1,16 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.36.1
-// 	protoc        v5.26.1
+// 	protoc-gen-go v1.34.0
```

Comment:
Does this version change look off (?). Followed the instructions [here](https://github.com/DataDog/datadog-agent/blob/main/pkg/proto/datadog/README.md#install)

Reply:
Hmm, looks like maybe your tools are slightly out-of-date?

Maybe try running `inv -e install-tools` and then rebuild the proto definitions again? 🤷🏻

Reply:
There is also a specific `inv install-protoc` (be careful it installs tools in `~/.local/bin` for some reason IIRC), I'm not sure if it runs as part of `install-tools`